schema/drafts/atac_schema.md

```diff
@@ -0,0 +1,511 @@
+      <td>Curator MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>str</code>. This MUST be the cell identifier. The value MUST be found in the <code>obs</code> index of the associated Dataset. Every <code>obs</code> index value of the associated Dataset MUST appear at least once in this column of the fragments file.
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+### fifth column
+
+<table><tbody>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>int</code>. This MUST be the total number of read pairs associated with this fragment. The value MUST be <code>1</code> or greater.
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+## scATAC-seq Asset: Fragments File index
+
+For every fragments file asset, CELLxGENE Discover MUST generate a <a href="https://www.htslib.org/doc/tabix.html">tabix</a> index of the fragment intervals from the fragments file. The file name MUST be the name of the corresponding fragments file appended with `.tbi`.
+
+## `uns` (Dataset Metadata)
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>peak_grouping</td>
+    </tr>
+    <tr>
+      <th>Annotation</th>
+      <td>Curator MAY annotate if the Dataset has a fragments file asset; otherwise, this key MUST NOT be present.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>
+          <code>str</code>. The value MUST match a key in <code>obs</code>. If annotated, genome track assets MUST be submitted. The following columns MUST NOT be specified:
+      <ul>
+        <li>assay_ontology_term_id</li>
+        <li>cell_type_ontology_term_id</li>
+        <li>development_stage_ontology_term_id</li>
+        <li>disease_ontology_term_id</li>
+        <li>organism_ontology_term_id</li>
+        <li>self_reported_ethnicity_ontology_term_id</li>
+        <li>sex_ontology_term_id</li>
+        <li>tissue_ontology_term_id</li>
+      </ul>
+      Instead specify the corresponding Discover column such as <code>cell_type</code>.<br><br>
+        </td>
+    </tr>
+</tbody></table>
+
+## scATAC-seq Asset: Genome Track
+
+If <code>uns['peak_grouping']</code> is annotated, there MUST be exactly one genome track asset submitted for each unique value in the obs column specified as determined by <code>anndata.obs.{peak_grouping_column}.unique()</code>. Otherwise, this MUST NOT be submitted.
+
+Asset file specifications TBD based on the visualization solution. Accepting <a href="https://genome.ucsc.edu/goldenpath/help/bigWig.html">.bigWig format</a> is a requirement.
+
+## Chromosome Tables
+
+As determined by the reference assembly used by the gene annotation versions pinned for this version of the schema. Only chromosomes or scaffolds that have at least one gene feature present are included.
+
+### <a href="https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_44/GRCh38.primary_assembly.genome.fa.gz">human (GRCh38.p14)</a>
+
+<table>
```

Comment:
Is this the expected order that chromosome should be sorted by? When doing a alphabetical sort I get something very different.

```
['chrY',
 'chrX',
 'chrM',
 'chr9',
 'chr8',
 'chr7',
 'chr6',
 'chr5',
 'chr4',
 'chr3',
 'chr22',
 'chr21',
 'chr20',
 'chr2',
 'chr19',
 'chr18',
 'chr17',
 'chr16',
 'chr15',
 'chr14',
 'chr13',
 'chr12',
 'chr11',
 'chr10',
 'chr1',
 'KI270750.1',
 'KI270744.1',
 'KI270734.1',
 'KI270733.1',
 'KI270731.1',
 'KI270728.1',
 'KI270727.1',
 'KI270726.1',
 'KI270721.1',
 'KI270713.1',
 'KI270711.1',
 'KI270442.1',
 'GL000225.1',
 'GL000220.1',
 'GL000219.1',
 'GL000218.1',
 'GL000216.2',
 'GL000213.1',
 'GL000205.2',
 'GL000195.1',
 'GL000194.1',
 'GL000009.2']
```

Reply:
NVM, this depend on the ordering that tabix expects.